README.md

```diff
@@ -182,7 +182,19 @@ Open [http://localhost:6006](http://localhost:6006) to view it in the browser.
 Exports Storybook as a static app to `storybook-static` folder.<br>
 
 - `npm test`
-Launches the test runner in the interactive watch mode executing `react-scripts-ts test -u --env=jsdom --coverage`.
+Launches test runner executing `react-scripts-ts test --env=jsdom --coverage`.
+
+- `test:watchAll:silent`
+Launches test runner watching all files exposing some details executing `react-scripts-ts test --env=jsdom --watchAll --silent --verbose`.
+
+- `test:watchAll:loud`
```

Comment:
Can you explain what the difference is between `silent` and `loud`?

Reply:
`test:watchChanged:silent` and `test:watchAll:silent` npm scripts use Jest CLI option [--silent](https://jestjs.io/docs/en/cli#silent) and [--verbose](https://jestjs.io/docs/en/cli#verbose) to get the following output: 

<img width="307" alt="screen shot 2018-09-20 at 11 55 47" src="https://user-images.githubusercontent.com/28991521/45837471-b4c88780-bd0f-11e8-9d6e-b0a535895a56.png">

whereas `test:watchChanged:loud` and `test:watchAll:loud` npm scripts only use Jest CLI option [--verbose](https://jestjs.io/docs/en/cli#verbose) to get this one: 

<img width="403" alt="screen shot 2018-09-20 at 19 57 23" src="https://user-images.githubusercontent.com/28991521/45837505-cdd13880-bd0f-11e8-812e-e612ebdff797.png">

Reply:
This is extremely useful! My only question is, could we not use the same names as Jest? So silent & verbose instead of silent & loud. Or is there a specific reason not to do that?

Reply:
I am fine with `verbose` instead of `loud`. Changed accordingly.